api.swagger.yaml

```diff
@@ -17659,6 +17758,19 @@ components:
           description: The location's email address.
           type: string
           format: email
+    location-any-schema:
+      oneOf:
```

Comment:
https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/ca1474c26ae66b7336b8943e1dda01d9775d65af/api.swagger.yaml

(no example available) showing, is it possible to just return the first schema?

Reply:
I think our version of swagger UI needs a bump. It looks fine in editor and doign this:
docker pull swaggerapi/swagger-ui
docker run -p 8080:8080 swaggerapi/swagger-ui

http://127.0.0.1:8080/
https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/INVGOV-13388.SWAGGER-5.17/api.swagger.yaml